pyqtgraph/parametertree/parameterTypes/basetypes.py

```diff
@@ -319,24 +319,37 @@ def __init__(self, param, depth):
         self.optsChanged(self.param, self.param.opts)
 
     def updateDepth(self, depth):
-        ## Change item's appearance based on its depth in the tree
-        ## This allows highest-level groups to be displayed more prominently.
-        if depth == 0:
-            for c in [0,1]:
-                self.setBackground(c, QtGui.QBrush(QtGui.QColor(100,100,100)))
-                self.setForeground(c, QtGui.QBrush(QtGui.QColor(220,220,255)))
-                font = self.font(c)
-                font.setBold(True)
+        """
+        Change item's appearance based on its depth in the tree
+        This allows highest-level groups to be displayed more prominently.
+        """
+        app = QtWidgets.QApplication.instance()
+        palette = app.palette()
+        textColor = palette.text().color()
+        backgroundColor = palette.button().color().darker(110)
+        altBackgroundColor = palette.dark().color()
+
+        # if stylesheet and 'QDarkStyleSheet' in stylesheet:
+        if app.styleSheet(): # Assume the stylesheet is qdarkstyle
```

Comment:
Although ntjess already pointed it out:
This seems like an overly broad condition to check. I do run a global stylesheet on my apps, and I might not be the only one. Having some elements of the app switch color as soon as you apply ANY stylesheet, even a completely empty one, seems like a good way to give a major headache to someone, somewhere, sometime 😆

Reply:
I've made the changes to this already but haven't pushed a new commit yet. The code now checks for both a light and dark version of the QDarkStyle stylesheet.